geopandas/tests/test_plotting.py

```diff
@@ -846,8 +879,17 @@ def test_style_kwargs_linewidth_listlike(self):
     def test_style_kwargs_alpha(self):
         ax = self.df.plot(alpha=0.7)
         np.testing.assert_array_equal([0.7], ax.collections[0].get_alpha())
-        with pytest.raises(TypeError):  # no list allowed for alpha
-            ax = self.df.plot(alpha=[0.7, 0.2, 0.9])
+        # TODO splitting array-like arguments for the different plot types
+        # is not yet supported
+        # try:
+        #     ax = self.df.plot(alpha=[0.7, 0.2, 0.9])
```

Comment:
This was failing because it is actually not yet supported, see https://github.com/geopandas/geopandas/issues/1379